docs/release_notes/release_notes.md

```diff
@@ -14,7 +14,27 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Features
 
+### Scenario Installer
+
+### OSTree Content
+
+### Lazy Sync
+
+### Docker V2
+
+### Mirror On Sync
+
+### Remote Execution Integration
```

Comment:
I is possible now to use Foreman remote execution plugin to execute the content actions (install package, errata etc.). This acts as an alternative to the current katello-agent capabilities and allows to execute the actions over SSH (or other providers that will by the remote execution plugin later). It also allows to customize the script to be run when executing the content action, as well as some advanced execution options (delayed run, recurring, splay time etc.)